docs(gcp): add uninstall and restore deleted roles section to getting started page (#34)
openclarity/docs.openclarity.io: content/docs/vmclarity/getting-started/deploy-gcp/_index.md

   ![VMClarity UI Dashboard](/img/vmclarity-ui-1.png)

1. Complete the {{% xref "/docs/vmclarity/getting-started/first-tasks/_index.md" %}}.

## Uninstall VMClarity

1. You can uninstall VMClarity using the gcloud manager.

   ```shell
   gcloud deployment-manager deployments delete <vmclarity deployment name>
   ```

## Restore deleted roles

> 1. Go to the roles tab in the [IAM & Admin page](https://console.cloud.google.com/iam-admin/roles) on GCP and search for VMClarity in the filter input.

Reviewer: ```suggestion
1.  On the [IAM & Admin page](https://console.cloud.google.com/iam-admin/roles) on GCP, open the **Roles tab**, then search for VMClarity in the filter input.
```
